### DISCUSSION TITLE
[Resolved] Unable to change production branch

### DISCUSSION THREAD

**Discussion by twhitson** [vercel/community#2755] [Bug Report]
We are unable to change the "production branch" on any of our projects. Typing in the name of the new production branch and clicking "Save" does nothing. This is happening to users with both "Member" and "Owner" permissions.

### Page to Investigate

https://vercel.com/{org}/{proj}/settings/git

### Steps to Reproduce

- Scroll down to "Production branch"
- Set a new branch name
- Click "Save"

#### Expected result:
- Production branch changes

#### Actual result:
- Nothing happens

**Comment by sukhminderx** [comment]
Observed this error in console. I too am facing same issue:

git-82130ca19448b502.js:1 Uncaught (in promise) TypeError: e.preventDefault is not a function
    at git-82130ca19448b502.js:1:33013


  GET https://vercel.com/api/v1/projects/ggrreyegdsvjsvdsjvdsjvgdvhI/production-deployment 404

**Comment by amyegan** [comment]
Thank you for reporting this. I confirmed that I'm also running into this problem. Asking the team for help investigating it now.

**Reply by amyegan** [reply]
The team found the source and addressed the problem. I confirmed it is working for me now. Thanks again for letting us know!

**Reply by shivam-jainn** [reply]
It's still not working

**Reply by amyegan** [reply]
Hi @shivam-jainn. The original problem was fixed back in June of last year. I just confirmed that I am still able to change the production branch for my projects. 

Don't forget that the older production deployment stays in place until a new deployment is started from the new production branch. Meaning the change is not noticeable until after a new commit is pushed to the new prod branch.

If you're still having trouble with this, please [start a new discussion](https://github.com/vercel/community/discussions/new) with details about your specific project. Be sure to share detailed reproduction steps or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) if you have one available.
